### ISSUE TITLE
Add TOML frontmatter syntax

### ISSUE THREAD

**Issue by razcore-rad** [SpaceVim/vim-markdown#1]
How can we add this? I tried following the code used to add YAML highlight, but I'm not really an expert on `vim` script and it will probably take me a bunch just to figure this out. I'm not even sure where `syntax/yaml.vim` comes from. I tried searching for it but it's not part of this repo, there's no `lang#yaml` layer I'm aware of either so I'm not sure.

Any help would be appreciated! Thanks

In other words I tried this:

```vim
unlet b:current_syntax
syntax include @Yaml syntax/yaml.vim
syntax region yamlFrontmatter start=/\%^---$/ end=/^---$/ keepend contains=@Yaml
syntax include @Toml syntax/toml.vim
syntax region tomlFrontmatter start=/\%^+++$/ end=/^+++$/ keepend contains=@Toml
let b:current_syntax = "markdown"
```

But doesn't look like it's working.
